allium/lib/aroileaders.py

```
@@ -1021,435 +995,333 @@ def _rank_operators(aroi_operators):
+    'network_veterans': _veteran_extras,
+    'reliability_masters': _score_extras,
+    'legacy_titans': _score_extras,
+    'bandwidth_masters': _score_extras,
+    'bandwidth_legends': _score_extras,
+    'ipv4_leaders': _ip_extras,
+    'ipv6_leaders': _ip_extras,
+    'validated_relays': _validated_extras,
+}
+
+
 def _format_leaderboard_entries(leaderboards, aroi_operators, relays_instance):
```

Ruff (0.15.20)

[warning] Missing return type annotation for private function `_format_leaderboard_entries`

(ANN202)